tests/integration/mibs/memory_test.py

```diff
@@ -0,0 +1,100 @@
+from itertools import cycle
+from shutil import which
+import subprocess
+import time
+import io
+import os
+
+import pytest
+import pytest_twisted
+
+from nav.ipdevpoll.snmp import snmpprotocol, AgentProxy
+from nav.ipdevpoll.snmp.common import SNMPParameters
+from nav.mibs import cisco_memory_pool_mib, juniper_mib, netswitch_mib
+
+
+ports = cycle(snmpprotocol.port() for i in range(50))
+
+
+@pytest.fixture(scope='session')
+def snmpsim():
+    snmpsimd = which('snmpsimd.py')
+    assert snmpsimd, "Could not find snmpsimd.py"
+    workspace = os.getenv('WORKSPACE', os.getenv('HOME', '/source'))
+    proc = subprocess.Popen(
+        [
+            snmpsimd,
+            '--data-dir={}/tests/integration/snmp_fixtures'.format(workspace),
+            '--log-level=error',
+            '--agent-udpv4-endpoint=127.0.0.1:1024',
+        ],
+        env={'HOME': workspace},
+    )
+
+    while not _lookfor('0100007F:0400', '/proc/net/udp'):
+        print("Still waiting for snmpsimd to listen for queries")
+        proc.poll()
+        time.sleep(0.1)
+
+    yield
+    proc.kill()
+
+
+def _lookfor(string, filename):
+    """Very simple grep-like function"""
+    data = io.open(filename, 'r', encoding='utf-8').read()
+    return string in data
+
+
+@pytest.fixture()
+def snmp_agent_proxy(snmpsim):
+    port = next(ports)
+    agent = AgentProxy(
+        '127.0.0.1',
+        1024,
+        community='placeholder',
+        snmpVersion='v2c',
+        protocol=port.protocol,
+        snmp_parameters=SNMPParameters(timeout=1, max_repetitions=5, throttle_delay=0),
+    )
+    return agent
```

Comment:
This fixture is also redundant - it's a verbatim copy of `sensors_test.snmp_agent_proxy()`, which should be moved to `conftest.py` for proper re-use here.